### PR TITLE
Add setting Autohide Template Elements

### PR DESCRIPTION
**Pull request by dev7355608**
#78

### CODE REVIEW

File: tokenmagic/lang/en.json [hunk 1]
```diff
@@ -37,6 +37,8 @@
   "TMFX.settings.autoTemplateEnabled.hint": "Disabling this option will disable all automatic template effects. NOTE: Only affects newly placed templates.",
   "TMFX.settings.defaultTemplateOnHover.name": "Default template grid on hover",
   "TMFX.settings.defaultTemplateOnHover.hint": "Display the default template grid only when hovering on the template layer.",
+  "TMFX.settings.autohideTemplateElements.name": "Autohide template elements",
```
Comment (Feu-Secret) [inline tokenmagic/lang/en.json:40]: A minor remark, I may prefer : "Auto hide template elements"

**Comment by dev7355608** [comment]
Changed the hover behavior for grid highlights as requested in #82.

**Comment by Feu-Secret** [comment]
Nicely done!

**Comment by dev7355608** [comment]
The localization files need to be updated for languages other than english:
- update `defaultTemplateOnHover.hint` (remove the "on the template layer" part)
- add  `autohideTemplateElements.name` & `autohideTemplateElements.hint`

**Comment by dev7355608** [comment]
Control icons are no longer shown to users that cannot control them.

**Comment by Feu-Secret** [comment]
> 
> 
> The localization files need to be updated for languages other than english:
> 
>     * update `defaultTemplateOnHover.hint` (remove the "on the template layer" part)
> 
>     * add  `autohideTemplateElements.name` & `autohideTemplateElements.hint`

I'll update French localization.

I realized that TMFX don't have a German translation 😺 

FYI : @lozalojo @zimm44 @drdwing (there is also the new `automatic framerate` option)

**Comment by lozalojo** [comment]
Thanks for mentioning, I will take a look tomorrow,

**Review by Feu-Secret — APPROVED** [review]

**Pull request merged**